src/main.ts

```diff
@@ -28,6 +28,14 @@ async function run(): Promise<void> {
     const authProvider = core.getInput('use_auth_provider');
     const useInternalIp = core.getInput('use_internal_ip');
 
+    // Add warning if using credentials
+    if (credentials) {
+      core.warning(
+        '"credentials" input has been deprecated and will be removed in a future release. ' +
```

Comment:
I don't know if we should say "will be removed in a future release". It would be better to highlight the benefits of using the `auth` action for now, and then add the "deprecation" in a few releases.

Reply:
Removed that bit- I also have a link in there that points to the relevant section in the main readme which expands on the benefits of using `auth`.

Reply:
Okay - I'm just a little worried about using the "deprecated" word here.

Reply:
Since we plan to remove this at some point, I think `deprecated` would encourage more folks to use auth. Happy to switch if you have anything particular in mind.